tests/unit/test_commons.h

```diff
@@ -151,9 +156,7 @@ struct ConvertTupleType<std::tuple<Args...>,Tuple2> {
 // Factory for constructed generated type param list for testing::Types
 template <typename TypeList, typename ValueT>
 struct TestFactory {
-  static constexpr auto const tup_size = std::tuple_size<TypeList>::value;
-  static constexpr auto const I = std::make_index_sequence<tup_size>{};
-  using ResultTupleType = typename SpliceMeta<TypeList,ValueT>::ResultTupleType;
+  using ResultTupleType = typename SpliceMeta<ValueT, TypeList>::ResultTupleType;
```

Comment:
warning spotted in this file (not related to this PR):
```
/checkpoint/tests/unit/test_commons.h:180:28: warning: moving a local object in a return statement prevents copy elision [-Wpessimizing-move]
/checkpoint/tests/unit/test_commons.h:180:28: note: remove 'std::move' call
```

Reply:
Fixed